f/OPC-560-encoding-profiles-for-zoom separate file

phoebem opened the pull request:
Zoom encoding profiles in a sep file for less clutter

phoebem committed:
f/OPC-560-encoding-profiles-for-zoom separate file

Zoom encoding profiles in a sep file for less clutter

phoebem committed:
f/OPC-560-encoding-profiles-for-zoom separate file

Zoom encoding profiles in a sep file for less clutter

karendolan reviewed (approved):
Looks good. can you link the companion Bitbucket pull so this doesn't go in without it.

phoebem committed:
added 53 30fps hls profiles

phoebem committed:
added 53 30fps hls profiles

lbjay committed:
pin chef-config version

in chef-config v16.5.77 a syntax change was introduced that is only
compatible with ruby >= 2.6

phoebem committed:
Merge branch 'f/OPC-560-encoding-profiles-for-zoom' of https://github.com/harvard-dce/mh-opsworks-recipes into f/OPC-560-encoding-profiles-for-zoom
